pkg/apis/serving/v1alpha1/kfservice_types.go

```diff
@@ -99,7 +100,7 @@ type CustomSpec struct {
 // KFServiceStatus defines the observed state of KFService
 type KFServiceStatus struct {
 	duckv1beta1.Status `json:",inline"`
-	URL                string                  `json:"url,omitempty"`
+	URL                *apis.URL               `json:"url,omitempty"`
```

Comment:
Looks knative URL is just an alias of net.URL(https://github.com/knative/pkg/blob/master/apis/url.go#L28), also we require the types to have `+k8s:openapi-gen=true` annotation now to generate python sdk and it looks that `net.URL` does not have the annotation. @jinchihe Can we double check if this is an issue ?

Reply:
@yuzisun @johnugeorge Yes, seems that's a problem. But I think we can go ahead to upgrade knative, I will try to upgrade openapi and sdk once the PR merged. I will log a ticket to trace that, thanks.